boards/arm/v2m_musca/v2m_musca-secure.dtsi

```diff
@@ -0,0 +1,59 @@
+/ {
+	flash0: flash@10200000 {
+		/* QSPI flash */
+		reg = <0x10200000 0x800000>;
+	};
+
+	sram0: memory@30000000 {
```

Comment:
Is this correct? SRAM starting at 0x3....?

Reply:
0x30000000 is the address in secure map for internal SRAM